buildout-mercator.cfg

```diff
@@ -9,6 +9,7 @@ dump-requirements-file = ${make_wheels:platform}-requirements.txt
 frontend.static_dir = src/mercator/mercator/build
 frontend_package_name = mercator
 backend_package_name = adhocracy_mercator
+frontend.profile_images_enabled = false
```

Comment:
It is, according to an email conversation I had with him (as long as @rukala is Rouven Brüess)